.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,73 @@
+name: 🐞 Bug
+description: Report a bug encountered while using prometheus-operator
+title: "[BUG] <title>"
+labels: ["kind/bug", "needs-triage"]
+body:
+- type: checkboxes
```

Comment:
I'm not convinced that it will avoid duplicates. But we can have a note or something similar telling people to look for existing issues.

Reply:
right it just ask as reminder :)

eg: https://github.com/slashpai/gh-test-repo/issues/new?assignees=&labels=kind%2Fbug%2Cneeds-triage&projects=&template=bug.yaml&title=%3Ctitle%3E